Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMLearnConfigWizardPage.xaml.cs

```diff
@@ -147,6 +187,26 @@ private void SetAutoMapElementTypesGridView()
             xAutoMapElementTypesGrid.InitViewItems();
         }
 
+        private void SetAutoMapElementLocatorsGridView()
+        {
+            GridViewDef defView = new GridViewDef(GridViewDef.DefaultViewName);
+            defView.GridColsView = new ObservableList<GridColView>();
+            defView.GridColsView.Add(new GridColView() { Field = nameof(ElementLocator.Active), WidthWeight = 8, MaxWidth = 50, HorizontalAlignment = System.Windows.HorizontalAlignment.Center, StyleType = GridColView.eGridColStyleType.CheckBox });
+            List<GingerCore.General.ComboEnumItem> locateByList = GingerCore.General.GetEnumValuesForCombo(typeof(eLocateBy));
+
+
+            GingerCore.General.ComboEnumItem comboItem = locateByList.Where(x => ((eLocateBy)x.Value) == eLocateBy.POMElement).FirstOrDefault();
+            if (comboItem != null)
+                locateByList.Remove(comboItem);
+
+            defView.GridColsView.Add(new GridColView() { Field = nameof(ElementLocator.LocateBy), Header = "Locate By", WidthWeight = 25, StyleType = GridColView.eGridColStyleType.ComboBox, CellValuesList = locateByList, });
```

Comment:
Locate By column not really need to be Combo because we not supposed to show it as drop list, because user should not be able to modify it, it supposed to be read only.
either remove the Combo and set it as string or not but make sure it is read only cell